sdk/Cargo.toml

```diff
@@ -138,20 +138,19 @@ tokio = { version = "1.32.0", default-features = false, features = [
 [features]
 default = ["client", "wallet", "tls"]
 
-irc_27 = ["url", "serde"]
-irc_30 = ["url", "serde"]
-ledger_nano = ["iota-ledger-nano"]
-mqtt = ["std", "regex", "rumqttc", "dep:once_cell"]
+irc_27 = ["serde", "dep:url"]
+irc_30 = ["serde", "dep:url"]
+ledger_nano = ["dep:iota-ledger-nano"]
+mqtt = ["std", "tls", "dep:regex", "dep:rumqttc", "dep:once_cell"]
 participation = ["storage"]
-rand = ["dep:rand"]
-rocksdb = ["dep:rocksdb", "storage"]
+rocksdb = ["storage", "dep:rocksdb"]
 serde = [
-    "serde_repr",
     "hashbrown/serde",
     "packable/serde",
     "primitive-types/serde_no_std",
     "iota-crypto/serde",
     "zeroize?/serde",
```

Comment:
Just wondering, can you do `"dep:zeroize?/serde"`?

Reply:
I don't think so